lib/TInitialState.php

```diff
@@ -93,6 +93,11 @@ protected function publishInitialStateForUser(IUser $user, IRootFolder $rootFold
 			!$this->talkConfig->isNotAllowedToCreateConversations($user)
 		);
 
+		$this->initialState->provideInitialState(
+			'calendar_enabled',
+			$appManager->isEnabledForUser('calendar', $user)
+		);
```

Comment:
I'm still not a fun of adding a new incorrect usage of the initial state...
Fine with it, if mobile clients don't need it.